core/templates/dev/head/domain/topic/TopicUpdateServiceSpec.ts

```diff
@@ -164,62 +136,26 @@ describe('Topic update service', function() {
   it('should create a proper backend change dict for removing an additional ' +
     'story id',
   function() {
-    TopicUpdateService.removeAdditionalStoryId(_sampleTopic, 'story_2');
+    TopicUpdateService.removeAdditionalStory(_sampleTopic, 'story_2');
     expect(UndoRedoService.getCommittableChangeList()).toEqual([{
-      cmd: 'update_topic_property',
-      property_name: 'additional_story_ids',
-      new_value: [],
-      old_value: ['story_2']
+      cmd: 'delete_additional_story',
+      story_id: 'story_2'
     }]);
   });
 
   it('should not create a backend change dict for removing an additional ' +
     'story id when an error is encountered',
   function() {
     expect(function() {
-      TopicUpdateService.removeAdditionalStoryId(_sampleTopic, 'story_5');
-    }).toThrow();
-    expect(UndoRedoService.getCommittableChangeList()).toEqual([]);
-  });
-
-  it('should add/remove a canonical story id to/from a topic',
```

Comment:
Ditto -- not sure I understand removal of these tests; it probably relates to the question of removing the "add" functions.

Reply:
Yeah, it's because I removed the add story functions.